### DISCUSSION TITLE
无法注册！！！！

### DISCUSSION THREAD

**Discussion by s-019** [walinejs/waline#704] [Q&A] [answered]
如图所示
![IMG_20211212_115927](https://user-images.githubusercontent.com/77848793/145699737-37896dec-5246-4d9f-9f80-2e285c42a369.jpg)

谢谢

看下是啥原因

**Comment by s-019** [comment]
我是按照快速开始搞的，不知道是啥原因

**Comment by s-019** [comment]
我LEADCLOUD用了是国际版的

**Comment by s-019** [comment]
感谢

**Comment by s-019** [comment]
快点回

**Reply by ghost** [reply]
请问解决了吗？我也有这个问题

**Comment by Mister-Hope** [comment, marked as answer]
并没有提供任何必要信息，甚至一个完整的网址。
